python/rikai/contrib/torch/transforms/utils.py

```diff
@@ -0,0 +1,21 @@
+#  Copyright 2021 Rikai Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+
+from rikai.types.vision import Image
+
+
+def uri_to_pil(uri):
```

Comment:
@sadhen FYI if you're wondering about use cases for the spark issue you're working on. This is a good one.